glitter/integration.py

```diff
@@ -0,0 +1,62 @@
+from django.conf import settings
+from importlib import import_module
+
+
+class GlitterApp(object):
+    """
+    A simple class to store Glitter App config options.
+    """
+    def __init__(self, name, url_conf, namespace=None):
+        self.name = name
+        self.url_conf = url_conf
+        self.namespace = namespace
```

Comment:
Can we add `self.sort_order` or this has been implemented to be able to sort the navigation?